app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -511,4 +516,61 @@ public Snackbar showSnackbarAbovePlayer(CharSequence text, int duration) {
     public Snackbar showSnackbarAbovePlayer(int text, int duration) {
         return showSnackbarAbovePlayer(getResources().getText(text), duration);
     }
+
+    //Hardware keyboard support
+    @Override
+    public boolean onKeyUp(int keyCode, KeyEvent event) {
+        Intent serviceIntent = new Intent(this, PlaybackService.class);
+        serviceIntent.putExtra(MediaButtonReceiver.EXTRA_HARDWAREBUTTON, true);
+
+        AudioManager audioManager = (AudioManager) getSystemService(AUDIO_SERVICE);
+        Integer customKeyCode = null;
+
+        switch (keyCode) {
+            case KeyEvent.KEYCODE_ENTER: //Fallthrough
+            case KeyEvent.KEYCODE_SPACE:
+                //Play/Pause
+                customKeyCode = KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE;
+                break;
+            case KeyEvent.KEYCODE_DPAD_LEFT:
+                //Go Back
+                customKeyCode = KeyEvent.KEYCODE_MEDIA_REWIND;
+                break;
+            case KeyEvent.KEYCODE_DPAD_RIGHT:
+                //Go Forward
+                customKeyCode = KeyEvent.KEYCODE_MEDIA_FAST_FORWARD;
+                break;
+            case KeyEvent.KEYCODE_M:
+                //Mute/Unmute
```

Comment:
Hmm, I think I would skip the mute functionality. For Audio, it is kind of pointless.

Reply:
Yes, I agree this is a personal decision but as most of the users use AntennaPod without an hardware keyboard and @alexanderadam wants a mute function, I think this should stay implemented, anyway.

Reply:
Ah sorry! @ByteHamster you're absolutely right, this was just an example. I copied these shortcuts from somewhere.

@asdoi thank you so much for working on this! :pray:

Reply:
Ok. I now removed it for audio podcasts (`MainActivty`) should I also remove it for videos (`VideoPlayerActivity`)